C_routing.md

```diff
@@ -619,12 +592,18 @@ We have come quite a long way in this guide without talking about one of the fir
 
 Remember in the [Overview Guide](http://www.phoenixframework.org/docs/overview) when we described plugs as being stacked and executable in a pre-determined order, like a pipeline? Now we're going to take a closer look at how these plug stacks work in the router.
 
-Pipelines are simply plugs stacked up together in a specific order and given a name. They allow us to customize behaviors and transformations related to the handling of requests. Phoenix provides default pipelines for common tasks, but it allows us to customize them, and it also allows us to create new pipelines to meet our needs.
+Pipelines are simply plugs stacked up together in a specific order and given a name. They allow us to customize behaviors and transformations related to the handling of requests. Phoenix provides us with some default pipelines for a number of common tasks. In turn we can customize them as well as creating new pipelines to meet our needs.
 
-A newly generated Phoenix application defines two pipelines, `:browser`, and `:api`. We'll get to those in a minute, but first we need to talk about the plug stack in the Endpoint.
+A newly generated Phoenix application defines two pipelines called `:browser` and `:api`. We'll get to those in a minute, but first we need to talk about the plug stack in the Endpoint plugs.
 
 #####The Endpoint Plugs
 
+The endpoint is the boundary where all requests start, thereby exposing the application interface to the underlying web services. It has three responsibilities:
```

Comment:
I don't think we can just use text verbatim from the docs without any attribution. We either need to re-write this section, add attribution, or remove it.